src/blog/2022/10/ff-docker-gcp.md

```diff
@@ -0,0 +1,180 @@
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null`
+  
+
+`sudo apt-get update`
+
+
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin`
+
+You can read a lot more detail about what each these commands actually do [here](https://docs.docker.com/engine/install/debian/).
+
+# Download FlowForge’s latest Docker build
+
+The next step is to get the codebase for FlowForge onto your VM, to do so you will need to run the following commands. Please note that we are working with our 0.10.0 build, you will need to update the version number in the commands below if you are working with a newer build.
+
+Install Wget so we can download FlowForge from Github.
+
+`sudo apt-get install wget`
+
+Use Wget to download the files we need.
+
+`wget https://github.com/flowforge/docker-compose/archive/refs/tags/v0.10.0.tar.gz`
+
+Make the directory where we will store FlowForge.
+
+`sudo mkdir /opt/flowforge`
+
+Uncompress FlowForge.
+
+`tar zxf v0.10.0.tar.gz`
+
+Copy Flowforge into the directy we created.
+
+`sudo cp -r docker-compose-0.10.0/ /opt/flowforge`
+
+You should now have all the code you need for FlowForge in the directory `/opt/flowforge/docker-compose-0.10.0`, it should look something like this.
+
+![List command output](images/8.png)
+
+# Configure FlowForge
+
+We can now configure FlowForge on your VM. We are going to need to edit two files. Firstly we need to switch into the directory where we just placed FlowForge.
+
+`cd /opt/flowforge/docker-compose-0.10.0`
+
+Then we need to edit the flowforge.yml file, we're using Nano to do that.
+
+`sudo nano /opt/flowforge/docker-compose-0.10.0/etc/flowforge.yml`
+
+At the top of the file you need to update the domain and base_url to match your domain
+
+![Domain config](images/9.png)
+
+Next we will need to edit the Email Configuration section to match your SMTP provider. Set enabled to true then add in the details provider by your email provider. For example in this case I am using our Google Workspace account.
+
+![Email config](images/10.png)
+
+Finally, you need to update the `public_url` for your mqtt broker to match your DNS record.
+
+![MQTT config](images/11.png)
+
+You can now save and close that file, in Nano you can do that by pressing ‘control x’ then ‘y’ then the Return key.
+
+Now we need to edit the `docker-compose.yml` file. We will use Nano again to do that.
+
+`sudo nano /opt/flowforge/docker-compose-0.10.0/docker-compose.yml`
+
+We need to edit the file to add in to the domain as follows.
+
+![Virtual Hosts config](images/12.png)
+
+Save and exit from that file, in Nano you can do that by pressing ‘control x’ then ‘y’ then the Return key.
+
+# Start FlowForge
+
+We are now ready to start up FlowForge for the first time, to do so we will use the following command.
+
+`sudo docker compose up -d`
```

Comment:
```suggestion
`sudo docker compose up -p flowforge -d`
```

I've updated the docs since you started to add this as it works round a problem when upgrading between versions

Reply:
```suggestion
`sudo docker compose -p flowforge up -d`
```